proxy/proxy.go

```diff
@@ -356,7 +356,10 @@ func (c *client) Receive(reader io.Reader) error {
 
 	switch msg := body.Message.(type) {
 	case *message.Options:
-		c.send(raw.Header, &message.Supported{Options: map[string][]string{"CQL_VERSION": {"3.0.0"}, "COMPRESSION": {}}})
+		c.send(raw.Header, &message.Supported{Options: map[string][]string{
+			"CQL_VERSION": {c.proxy.cluster.Info.CQLVersion},
```

Comment:
This is the fix.

Reply:
Ah interesting, in ZDM we are just forwarding this request to one of the clusters (which one is determined by a config setting). The only request that we generate a response for is `system.peers` and `system.local` queries, everything else is forwarded.